ticker/src/main/java/com/robinhood/ticker/TickerUtils.java

```diff
@@ -23,48 +23,13 @@
  * @author Jin Cao, Robinhood
  */
 public class TickerUtils {
-    public static final char EMPTY_CHAR = (char) 0;
-
-    /**
-     * @return a default ordering for all viewable ASCII characters. This list also special cases
-     *         space to be in front of the 0 and swap '.' with '/'. These special cases make
-     *         typical US currency animations intuitive.
-     */
-    public static char[] getDefaultListForUSCurrency() {
-        final int indexOf0 = (int) '0';
-        final int indexOfPeriod = (int) '.';
-        final int indexOfSlash = (int) '/';
-
-        final int start = 33;
-        final int end = 256;
-
-        final char[] charList = new char[end - start + 1];
-        for (int i = start; i < indexOf0; i++) {
-            charList[i - start] = (char) i;
-        }
-
-        // Special case EMPTY_CHAR and '.' <-> '/'
-        charList[indexOf0 - start] = EMPTY_CHAR;
-        charList[indexOfPeriod - start] = '/';
-        charList[indexOfSlash - start] = '.';
-
-        for (int i = indexOf0 + 1; i < end + 1; i++) {
-            charList[i - start] = (char) (i - 1);
-        }
+    static final char EMPTY_CHAR = (char) 0;
 
-        return charList;
+    public static String provideNumberList() {
+        return "0123456789";
     }
 
-    /**
-     * @return a default ordering for numbers.
-     */
-    public static char[] getDefaultNumberList() {
```

Comment:
We should update the README to reflect public API changes.

Reply:
good catch. and add an entry to the changelog.md